### DISCUSSION TITLE
Initial Multi Column Sorting on Mulitple Columns

### DISCUSSION THREAD

**Discussion by lgastmans** [Power-Components/livewire-powergrid#1612] [Ideas]
# ⚡ PowerGrid ⚡ Feature Request
The initial sort of the table can be set using:

    public string $sortField = 'column_name';
    public string $sortDirection = 'asc';

But I would like to have the initial sort on multiple columns, where the columns are related, as in:

    public string $sortField = 'collection_number, families.name, genuses.name';
    public string $sortDirection = 'asc';

**Comment by luanfreitasdev** [comment]
This is cool, if you want to open a PR feel free. However, you can use the "multi sort" feature that enables this according to the columns you defined to be "sortable()"

https://livewire-powergrid.com/table-features/sorting-data.html#multi-column-sorting
